test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -1,7 +1,7 @@
 PROJECT_NAME           = "SeqAn3"
 PROJECT_BRIEF          = "The Modern C++ library for sequence analysis."
 PROJECT_LOGO           = ${SEQAN3_DOXYGEN_SOURCE_DIR}/test/documentation/seqan_logo.png
-PROJECT_NUMBER         = ${SEQAN3_VERSION}
+PROJECT_NUMBER         = ${CPACK_PACKAGE_VERSION}
```

Comment:
I think we should keep `SEQAN3_VERSION`, because that will be populated by `find_package(seqan3)`